.github/workflows/build_and_deploy_hold.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and deploy with a hold state
+
+on:
+  push:
+    tags:
```

Comment:
What does this mean? What do we need to do to get this workflow to kick off?

Reply:
How its currently setup I think this will run automatically on any push. I think we'd want to change it so its to every push to main? 
```
on:
  push:
    branches:
      - main
```
Could also require a manual run: 
```
on:
  workflow_call:
```